README.md

```diff
@@ -7,9 +7,9 @@ This extension contains various components of the [Rust](https://www.rust-lang.o
 
 In order to use the (fast) [mold](https://github.com/rui314/mold) linker:
 
-1. Add `org.freedesktop.Sdk.Extension.llvm13` allong with this extension in order to get clang.
-2. Add `/usr/lib/sdk/llvm13/lib` in `append-ld-library-path` and `append-path`. See [llvm13 SDK extension readme](https://github.com/flathub/org.freedesktop.Sdk.Extension.llvm13) for more information.
-3. Add `config.toml` with the following content in `CARGO_HOME` directory:
+1. Add `org.freedesktop.Sdk.Extension.llvm13` along with this extension in order to get clang.
+2. Add `/usr/lib/sdk/llvm13/bin` to `append-path`. See [llvm13 SDK extension readme](https://github.com/flathub/org.freedesktop.Sdk.Extension.llvm13) for more information.
+3. Add `.cargo/config.toml` in the root of the repository with the following content:
```

Comment:
Ideally you want the config file to be only in your local `cargo_home` to avoid making everything go through mold regardless the environment

Reply:
That's what happens if I add it at the repo root, no?

Reply:
No, when you add it to the repo root, its treated as the configuration to be used for the whole repository, regardless if mold exists or not

Reply:
Let's go with this for now